.github/templates/main.yml

```diff
@@ -266,10 +276,40 @@ jobs:
         _: #@ template.replace(msbuild("Tests/Realm.Tests", TargetFramework="net461", RestoreConfigFile="${{ github.workspace }}/Tests/Test.NuGet.Config", UseRealmNupkgsWithVersion="${{ needs.build-packages.outputs.package_version }}" ))
```

Comment:
We can modify the `msbuild` function to return the entire node like:

```yaml
#@ def msbuild(projectPath, **properties):
#...
name: #@ "Build " + projectPath
run: #@ "msbuild ${{ github.workspace }}/" + projectPath + " " + target + " -p:Configuration=" + configuration + " -restore" + parsedProps
#@ end
```

which will then simplify the usage to:

```yaml
- #@ msBuild("Tests/Realm.Tests", ...)
```